components/Alias.tsx

```diff
@@ -25,11 +28,12 @@ const Alias = ({
   const toDisplay = useMemo(
     () =>
       state.aliases[address] === ""
```

Comment:
what if  `state.alises[address]` returns null? Maybe it's not possible, feel free to resolve if it's the case